internal/validators/types.go

```diff
@@ -35,6 +37,11 @@ type Metadata struct {
 	StaleTimestamp time.Time `json:"stale_timestamp"`
 }
 
+type Transport struct {
```

Comment:
Nitpick, I'm not sure Transport is a good name here.  "Transport" makes me think of a socket or an connection of some sort.  Maybe something like "ValidationEvent", "ValidationMessage".  I'm not sure.  Naming things is hard.  :)